src/Validator/ProgrammeDateTimeDifference.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Validator;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @Annotation
+ */
+class ProgrammeDateTimeDifference extends Constraint
+{
+    public string $message = 'Difference between programmes end date and start date should be greater or equal 
+                                with {{ minTimeMin }} minutes and lesser or equal with {{ maxTimeMin }} minutes';
```

Comment:
this is not the correct way to write a string on multiple lines. You need to use concatenation. If not, your message has a newline and a bunch of spaces/tabs which will look bad in the response

Reply:
Solved.